### ISSUE TITLE
Freezes and ctrl+alt+f3 won't go tty

### ISSUE THREAD

**Issue by omega16** [swaywm/sway#5576]
Random freeze some times, with vcl open or not but i was able to reproduce in less than 40 secs with vcl

Steps I take:
- Start pc
- Stop lightm service from tty
- Start sway from tty using `sway -d 2>~/sway.log`
- Open thunar and use to open as many vlc instances
- Ensure all vlc instances are can be viewed on screen to stress pc.
- Freeze.

I obtained the dump from `sudo coredumpctl gdb sway`

Here is config file and log
https://gist.github.com/omega16/63f6aa951601fd29f8ded22b6c013682

I have another log of previous attempt to catch, it took around 20 minutes and has 12000 lines, i can post it if you need it too.

**Comment by omega16** [comment]
I guess i do a mistake. I can't trigger freeze with that and the swaylog i provided don't seems to have an error, at least not compared to the previous catch.


So, I'm posting the previous attempt to catch the crash here
https://gist.github.com/omega16/086da5a554fc8455c5a4601722e0a046

I forgot to put sway version so 

`swaymsg -t get_version`
sway version v1.5

**Comment by duck-rh** [comment]
I am too experiencing random freeze with Sway 1.5. I may happen shortly after launch while exec-ed apps are being started and I rearrange a bit or after 30+ minutes. It happened while typing in Firefox, or Signal desktop, or playing a Wine game; all of these are XWayland apps so maybe it's related.

When this happen the whole compute freezes, no going back to the console. Attaching the log.

[sway__freeze.log](https://github.com/swaywm/sway/files/4977791/sway__freeze.log)

**Comment by omega16** [comment]
I don't know why, but yesterdar it works all day (like 16 hours) without problems, today it took 2 hours to crash 
I cut the log to just last minutes since it can't be fully loaded on gist.

https://gist.github.com/omega16/f063e440f706122143639752c32666fa

Some pointers on how to read log so i can send more usefull logs (by cutting all that don't matter) or some others tool to run would be appreciated.  Since crash is hard to reproduce (to me) I'm working all time with debug enabled (same as before, kill ligthdm and run sway -dV ).

**Comment by Xyene** [comment]
Your stack traces are missing debug symbols, so it's not really possible to tell what went wrong. You can find instructions on compiling sway from source [here](https://github.com/swaywm/sway/wiki/Development-Setup#compiling-as-a-subproject). Note that debug symbols are included in Meson builds by default. That way, when/if you are able to reproduce this issue, the coredump generated will be useful.

**Comment by duck-rh** [comment]
just to say I tried again 1.5 after a few libs were updated on my system (Debian unstable) and this past week I've not been able to experience this problem again. @omega16 can you still reproduce?

**Comment by omega16** [comment]
No, I have been runing by hand a sway with debug symbols and unless I mess around with config, sway won't crash.

**Comment by duck-rh** [comment]
Since last time I realized I forgot to reactivate `adaptive_sync`, which was the only change I made when switching to 1.5. I reenabled it and still no crash.

@omega16 unless you figure out which config change triggers the crash I guess noone will be able to help you. I would try changing them one by one, like one every day, and see how stable it is.
